crates/orderbook/src/api/get_order_status.rs

```diff
@@ -25,6 +28,13 @@ pub fn get_status(
                     super::error("OrderNotFound", "Order not located in database"),
                     StatusCode::NOT_FOUND,
                 ),
+                Err(err @ orderbook::Error::NotFound) => {
+                    tracing::error!(?err, "get_order_status");
+                    reply::with_status(
+                        super::error("NotFound", "Competition was not found"),
```

Comment:
While the error originates from not finding the competition from the user's perspective this message makes more sense.
```suggestion
                        super::error("NotFound", "Order status was not found"),
```

Reply:
Another comment resolved without any fix or explanation why it does not need be addressed.

The caller requested information about the status of an order. They might not even know what `Competition` you are talking about or care about implementation the associated implementation details that produce this specific error.

Reply:
Discussed and resolved in DMs